cli/cdsctl/encrypt.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/ovh/cds/cli"
+)
+
+var (
+	encryptCmd = cli.Command{
+		Name:  "encrypt",
+		Short: "Encrypt variable into your CDS project",
```

Comment:
missing long documentation for public doc (explain usecases for example)

Reply:
```


Create a secret variable:


    $ cdsctl encrypt MYPROJECT my-data my-super-secret-value
    mydata: 01234567890987654321

The command returns the value: `01234567890987654321`. You can use this value in a configuration file.

Example of use case: Import an environment with a secret.

Create an environment file to import :


    $ cat << EOF > your-environment.json
    name: your-environment
    values:
    aReadableVariable:
        type: string
        value: registry.ovh.net/engine/http2kafka
    APasswordVariable:
        type: password
        value: 01234567890987654321
    EOF


Then, import then environment:

    cds environment import MYPROJECT your-environment.json



```